### DISCUSSION TITLE
What Starter Kit Would You Like To See?

### DISCUSSION THREAD

**Discussion by CobyPear** [pantheon-systems/decoupled-kit-js#396] [Polls]
TL;DR: In what frameworks does our community see the most potential?

In the PHP CMS world, Pantheon has found success by striking a happy medium. We support WordPress and Drupal. We do not support every CMS that _could_ run on Pantheon infrastructure. Joomla, Craft, Laravel and others are not that different in their infrastructure requirements.

Both for the Front-End Sites product, and this Open Source Decoupled Kit initiative, we want to find a similar path. We will not put all of our eggs in the basket of one framework. But we don't expect web teams to be successful with us spreading efforts across every framework under the sun either.

\- Decoupled Kit Team
